ParProcCo/utils.py

```diff
@@ -2,13 +2,51 @@
 
 import logging
 import os
+import yaml
+
 from dataclasses import dataclass
 from datetime import datetime
 from pathlib import Path
-from typing import Dict, List, Optional, Set, Union
+from typing import Dict, List, Literal, Optional, Set, Union
 from yaml import YAMLObject, SafeLoader
 
 
+GPFS02 = [
```

Comment:
Relocate this beamline info in `msmapper-utils`'s cluster submit script as PPC should be independent of DLS